### DISCUSSION TITLE
Great work

### DISCUSSION THREAD

**Discussion by EATREVITPOOPCAD** [Nice3point/RevitExtensions#2] [General]
This is amazing! Thank you for making this.

![image](https://user-images.githubusercontent.com/91353212/149549742-d2ed345b-3c81-4a32-bf70-7d8c0393106b.png)

**Comment by Nice3point** [comment]
@EATREVITPOOPCAD thanks) if you have any ideas, feel free to write about them, we will make this damn Revit better
